app/views/trips/_trip.html.erb

```diff
@@ -39,6 +39,11 @@
     <%= trip.luggage %>
   </p>
 
+  <p>
+    <strong>Vehicle type:</strong>
+    <%= Vehicle.where(id: trip.vehicle_id).map { |elm| elm.vehicle_type } %>
```

Comment:
why? can't you simply do `trip.vehicle.type`?

Reply:
you are right. fixed!